egs/wsj/s5/steps/libs/common.py

```diff
@@ -284,6 +317,109 @@ def write_kaldi_matrix(output_file, matrix):
         f.write(" ]")
 
 
+def write_matrix_ascii(file_or_fd, mat, key=None):
```

Comment:
As a general thought, how hard would it be instead to swig Kaldi's own I/O functions from python?